vsphere/variables.libsonnet

```diff
@@ -0,0 +1,112 @@
+local g = import './g.libsonnet';
+local var = g.dashboard.variable;
+local commonlib = import 'common-lib/common/main.libsonnet';
+local utils = commonlib.utils;
+
+// Generates chained variables to use on on all dashboards
+{
+  new(this, varMetric):
```

Comment:
Your variables need formatted display names. I haven't done this in the new format, but Istio has some examples.